TODO.md

```diff
@@ -0,0 +1,45 @@
+# General
+
+* [ ] Write decent docs about our example implementation
+* [ ] Search for @todo
+* [ ] Check why we can't overrule the title-block from within a template
+* [ ] Logging, search for papertrail
+
+# Frontend
+
+* [ ] learn twig
+* [ ] assetic with grunt
+* [ ] upgrade HTML to bootstrap ...
+* [ ] buttons in header: correct width
+* [ ] Tijs: make Data.js work again
+* [ ] Tijs: make Locale.js work again
```

Comment:
Locale.js already works, could be removed from the checklist. (I think some other stuff could be ticked as well)